libs/SmartSync/SmartSync/Classes/Util/SFSmartSyncObjectUtils.m

```diff
@@ -32,6 +32,8 @@ @implementation SFSmartSyncObjectUtils
 + (void) initialize {
     utcDateFormatter = [NSDateFormatter new];
     isoDateFormatter = [NSDateFormatter new];
+    NSLocale *enUSPOSIXLocale = [NSLocale localeWithLocaleIdentifier:@"en_US_POSIX"];
+    [isoDateFormatter setLocale:enUSPOSIXLocale];
```

Comment:
I would have thought that setting the date format would be sufficient but you are right, we also need to set the locale. Also you need to set the time zone to UTC. Could you add:
```objective-c
[isoDateFormatter setTimeZone:[NSTimeZone timeZoneForSecondsFromGMT:0]];
```
For more info see: https://developer.apple.com/library/archive/qa/qa1480/_index.html


For more information see: https://developer.apple.com/library/archive/qa/qa1480/_index.html
Could you also

Reply:
I will do the `setTimeZone` in a separate PR. Thanks for your contribution.